### ISSUE TITLE
CLI Request: Add "--all" option

### ISSUE THREAD

**Issue by bdkjones** [browserslist/browserslist#297]
I'd like to be able to run a command like this:

    browserslist --all ">0.5%, last 2 versions, not dead"

And have browserslist return BOTH the list of supported browsers AND the coverage statistics for the US and world, all in one fell swoop. Right now, to get this information, I must run three separate commands:

    browserslist ">0.5%, last 2 versions, not dead"
    browserslist --coverage ">0.5%, last 2 versions, not dead"
    browserslist --coverage=us ">0.5%, last 2 versions, not dead"

I envision that the coverage information could simply be added to the bottom of the list of browsers. To maintain the standard output format:

    [browser_name] [space] [version string]

I would suggest formatting the last two "coverage" lines as:

    coverage_world [space] 90.34
    coverage_us [space] 87.65

This drops the extra text returned by `--coverage` and allows consumers of the output to still parse it as a newline-separated list of space-separated key-value pairs.

**Comment by bdkjones** [comment]
This proposal would also not break existing consumers of browserlist's output, because the extra two lines would not appear unless the `--all` flag is passed.

**Comment by ai** [comment]
Interesting. Can you tell me about your use case?

I am thinking about the more clear name of the argument. And I do not like to [lock some specific country](https://github.com/browserslist/browserslist/pull/298/files#diff-2cce40143051e25f811b56c79d619bf5R98).

**Comment by bdkjones** [comment]
I have a GUI app where I want to display both the supported browser list AND the overall coverage stats based on the user’s chosen setting. (Similar to http://browsersl.ist ) This app is not in a JavaScript environment, so I can’t use the api directly; I need to go through the command line.

I’d like to get all the info in one shot because that lets me update the UI in real-time as the user types their browser string. Otherwise, each update would require three separate runs of Browserslist.

I had considered a comma-separated list of country codes, such as:

    --all=us,uk,cn

Which would return that info as well as the global coverage. An alternative flag name might be --include-coverage, or similar, but that was close to the existing API, which I didn’t want to change.

**Comment by bdkjones** [comment]
I have a GUI app where I want to display both the supported browser list AND the overall coverage stats based on the user’s chosen setting. (Similar to http://browsersl.ist ) This app is not in a JavaScript environment, so I can’t use the api directly; I need to go through the command line.

I’d like to get all the info in one shot because that lets me update the UI in real-time as the user types their browser string. Otherwise, each update would require three separate runs of Browserslist. 

I had considered a comma-separated list of country codes, such as: 
    
    --all=us,uk,cn

Which would return that info as well as the global coverage. An alternative flag name might be 
`--include-coverage`, or similar, but that was close to the existing API, which I didn’t want to change.

**Comment by bdkjones** [comment]
And thanks for considering the request! It would really help me and I don’t want to maintain my own fork. If we can come up with a better flag name, I’ll modify the PR.

**Comment by ai** [comment]
Yeap, reasonable use case 👍

Maybe we should use JSON format there?

What do you think about `browserslist --json --coverage=en,ru,world`?

**Comment by bdkjones** [comment]
I'll try to knock it out next week. Thanks!

**Comment by ai** [comment]
`---coverage=x,x,x` support done 9fd90d8

**Comment by ai** [comment]
JSON output format was added:

```js
➤ browserslist --coverage=US,alt-AS,global defaults --json
{
  "browsers": [
    "and_chr 69",
    "and_ff 62",
    "and_qq 1.2",
    "and_uc 11.8",
    "android 67",
    "android 4.4.3-4.4.4",
    "android 4.4",
    "baidu 7.12",
    "chrome 69",
    "chrome 68",
    "chrome 67",
    "chrome 63",
    "chrome 49",
    "edge 17",
    "edge 16",
    "firefox 62",
    "firefox 61",
    "firefox 60",
    "ie 11",
    "ie_mob 11",
    "ios_saf 11.3-11.4",
    "ios_saf 11.0-11.2",
    "ios_saf 10.3",
    "op_mini all",
    "op_mob 46",
    "opera 55",
    "opera 54",
    "safari 12",
    "safari 11.1",
    "samsung 7.2",
    "samsung 6.2",
    "samsung 4"
  ],
  "coverage": {
    "US": 89.24,
    "alt-AS": 26.92,
    "global": 89.39
  }
}
```

@bdkjones I am ready to release it when you will prove output format

**Comment by bdkjones** [comment]
Looks great to me! Thanks for knocking this out.

**Comment by ai** [comment]
@bdkjones I am writing release announce tweet. Is it for CodeKit?

**Comment by bdkjones** [comment]
I'll be using the feature in CodeKit for sure! I don't have any sort of "CodeKit" account on Twitter though. I just use my personal handle: @bdkjones

**Comment by ai** [comment]
Released in 4.2
